goreleaser.yml

```diff
@@ -142,27 +138,32 @@ dockers:
     - contrib/
 
 docker_manifests:
-  - name_template: 'aquasec/trivy:{{ .Version }}'
+  - name_template: 'devopps/trivy:{{ .Version }}'
     image_templates:
-    - 'aquasec/trivy:{{ .Version }}-amd64'
-    - 'aquasec/trivy:{{ .Version }}-arm64'
+    - 'devopps/trivy:{{ .Version }}-amd64'
+    - 'devopps/trivy:{{ .Version }}-arm64'
   - name_template: 'ghcr.io/aquasecurity/trivy:{{ .Version }}'
     image_templates:
-    - 'ghcr.io/aquasecurity/trivy:{{ .Version }}-amd64'
-    - 'ghcr.io/aquasecurity/trivy:{{ .Version }}-arm64'
-  - name_template: 'public.ecr.aws/aquasecurity/trivy:{{ .Version }}'
+    - 'ghcr.io/developer-guy/trivy:{{ .Version }}-amd64'
```

Comment:
Looks like it is your account.

Reply:
so sorry about that, I forgot to revert the changes that I've been made for testing purposes.